server/container/guice/protocols/jmap/src/test/java/org/apache/james/jmap/MailetPreconditionTest.java

```diff
@@ -191,5 +191,19 @@ void bccMailetCheckShouldNotThrowOnValidPair() throws Exception {
             assertThatCode(() -> MailetContainerModule.BCC_Check.check(pairs))
                 .doesNotThrowAnyException();
         }
+
+        @Test
+        void bccMailetCheckShouldNotThrowOnValidPairWithOtherNames() throws Exception {
```

Comment:
Edge case: Please add a test ensuring that removing `x-bcc-bug` header is not considered as valid.